Promtail Packages

Currently we don't have an easy way for anyone wanting to use standalone promtail to install it as a service on their host machine(s).

We should at least provide pre-built binaries, and ideally packages that can set up the services to run it at boot.

For starters we should provide .deb and .rpm packages.

I tried to build the Windows libraries with `go get` & `go build` but I got the following error:
```
+ GOOS=windows go get
# github.com/grafana/loki/vendor/github.com/cortexproject/cortex/pkg/chunk/cache
../../vendor/github.com/cortexproject/cortex/pkg/chunk/cache/diskcache.go:105:14: undefined: unix.Mmap
../../vendor/github.com/cortexproject/cortex/pkg/chunk/cache/diskcache.go:105:58: undefined: unix.PROT_READ
../../vendor/github.com/cortexproject/cortex/pkg/chunk/cache/diskcache.go:105:73: undefined: unix.PROT_WRITE
../../vendor/github.com/cortexproject/cortex/pkg/chunk/cache/diskcache.go:105:90: undefined: unix.MAP_SHARED
../../vendor/github.com/cortexproject/cortex/pkg/chunk/cache/diskcache.go:125:12: undefined: unix.Munmap
```

I tried on commit fa4f936fcbbad1568209864c0756b249769caa63

I'm puzzled. The chunk/diskcache components are used in the server, are they also needed in the promtail client?

--- a/docs/sources/installation/local.md
+++ b/docs/sources/installation/local.md
@@ -11,8 +11,23 @@ In order to log events with Grafana Loki, download and install both Promtail and
 
 The configuration specifies running Loki as a single binary.
 
-## Install
+## Install using APT or RPM package managers
 
+1. Add Granafa's [APT](https://apt.grafana.com/) or [RPM](https://rpm.grafana.com/)
+   package repository following the linked instructions.
+1. Install Loki and Promtail
+   1. Using `dnf`
+      ```
+      dnf update
+      dnf install loki promtail
+      ```
+   1. Using `apt-get`
+      ```
+      apt-get update
+      apt-get install loki promtail
+      ```
+
+## Install packages manaullypackages manaully
 1. Navigate to the [release page](https://github.com/grafana/loki/releases/).
 2. Scroll down to the Assets section under the version that you want to install.
 3. Download the Loki and Promtail .zip files that correspond to your system.

--- a/docs/sources/installation/local.md
+++ b/docs/sources/installation/local.md
@@ -27,7 +27,7 @@ The configuration specifies running Loki as a single binary.
       apt-get install loki promtail
       ```
 
-## Install packages manaullypackages manaully
+## Install manually
 1. Navigate to the [release page](https://github.com/grafana/loki/releases/).
 2. Scroll down to the Assets section under the version that you want to install.
 3. Download the Loki and Promtail .zip files that correspond to your system.

--- a/docs/sources/installation/local.md
+++ b/docs/sources/installation/local.md
@@ -11,7 +11,7 @@ In order to log events with Grafana Loki, download and install both Promtail and
 
 The configuration specifies running Loki as a single binary.
 
-## Install using APT or RPM package managers
+## Install using APT or RPM package manager
 
 1. Add Granafa's Advanced Package Tool [APT](https://apt.grafana.com/) or RPM Package Manager [RPM](https://rpm.grafana.com/)
    package repository following the linked instructions.

--- a/docs/sources/installation/local.md
+++ b/docs/sources/installation/local.md
@@ -13,7 +13,7 @@ The configuration specifies running Loki as a single binary.
 
 ## Install using APT or RPM package managers
 
-1. Add Granafa's [APT](https://apt.grafana.com/) or [RPM](https://rpm.grafana.com/)
+1. Add Granafa's Advanced Package Tool [APT](https://apt.grafana.com/) or RPM Package Manager [RPM](https://rpm.grafana.com/)
    package repository following the linked instructions.
 1. Install Loki and Promtail
    1. Using `dnf`

